book/src/lib/docker/test_helpers.md

```diff
@@ -0,0 +1,8 @@
+# Test helpers
+
+There two test helper containers:
+* Killgrave
+* Mockserver
+
+Both represent HTTP mocking solutions, with Mockserver used in k8s and Killgrave outside of it. Since both will soon
```

Comment:
If there is a ticket, PR etc - that will help here

Reply:
added